package.json

```diff
@@ -21,7 +21,7 @@
     "oclif.manifest.json"
   ],
   "bin": {
-    "balena": "./bin/balena"
+    "balena": "bin/balena"
```

Comment:
Was this change necessary for some specific reason? At least on Linux, the current working directory is sometimes not included in the executable search path, so attempting to execute `myApp` fails where `./myApp` succeeds.